app/models/core/UserAnswers.scala

```diff
@@ -49,7 +49,8 @@ final case class UserAnswers(
                               data: JsObject = Json.obj(),
                               progress : RegistrationStatus = NotStarted,
                               createdAt : LocalDateTime = LocalDateTime.now,
-                              internalAuthId :String
+                              internalAuthId :String,
+                              isTaxable: Boolean = false
```

Comment:
We should probably be default taxable to true?

If this is deployed and a user loads their answers from mongo they will become non-taxable which will break their journey and result them in getting the wrong confirmation page.